### PR TITLE
Updated docs for feature flags

### PR DESCRIPTION
**Pull request by ankit01-oss**

### CODE REVIEW

File: docs/operate/feature-flags.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Available Feature Flags
+
+`TIMESTAMP_SORT_FEATURE` and `DURATION_SORT_FEATURE` are enabled by default.
+
+- `TIMESTAMP_SORT_FEATURE`: When this feature is enabled, then SigNoz creates a materialized view table to optimize sorting spans/traces by timestamp at cost of increasing storage by 40-50% of `signoz_index_v2`
+
+- `DURATION_SORT_FEATURE`: When this feature is enabled, then SigNoz creates a projection over table to optimize sorting spans/traces by timestamp at cost of increasing storage by 40-50% of `signoz_index_v2`
+
+You can disable these features by adding configs to OTel Collector and Query Service.
```
Comment (makeavish) [inline docs/operate/feature-flags.md]: We should highlight this line:

> To toggle features, configs have to be added on both OTel Collector and Query Service

File: docs/operate/feature-flags.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+description: Learn how to use feature flags in SigNoz.
+---
+
+## Available Feature Flags
+
+`TIMESTAMP_SORT_FEATURE` and `DURATION_SORT_FEATURE` are enabled by default.
+
+- `TIMESTAMP_SORT_FEATURE`: When this feature is enabled, then SigNoz creates a materialized view table to optimize sorting spans/traces by timestamp at cost of increasing storage by 40-50% of `signoz_index_v2`
+
+- `DURATION_SORT_FEATURE`: When this feature is enabled, then SigNoz creates a projection over table to optimize sorting spans/traces by timestamp at cost of increasing storage by 40-50% of `signoz_index_v2`
```
Comment (makeavish) [inline docs/operate/feature-flags.md]: add 'table' at the end of the line to complete the sentence

File: docs/operate/feature-flags.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+id: feature-flags
+title: Feature Flags
+description: Learn how to use feature flags in SigNoz.
+---
+
+## Available Feature Flags
+
+`TIMESTAMP_SORT_FEATURE` and `DURATION_SORT_FEATURE` are enabled by default.
+
+- `TIMESTAMP_SORT_FEATURE`: When this feature is enabled, then SigNoz creates a materialized view table to optimize sorting spans/traces by timestamp at cost of increasing storage by 40-50% of `signoz_index_v2`
```
Comment (makeavish) [inline docs/operate/feature-flags.md]: add 'table' at the end of the line to complete the sentence

**Review by makeavish — CHANGES_REQUESTED** [review]

File: docs/operate/feature-flags.md (hunk 2 above)
Comment (ankit01-oss) [reply docs/operate/feature-flags.md]: Incorporated.

**Comment by ankit01-oss** [comment]
@makeavish merged.

**Pull request merged**